tests/integration/test_data/test_github-app_data_tests.integration.test_github_app.GithubTests.test_get_project.yaml

```diff
@@ -0,0 +1,85 @@
+        License","spdx_id":"MIT","url":"https://api.github.com/licenses/mit","node_id":"MDc6TGljZW5zZTEz"},"forks":12,"open_issues":22,"watchers":14,"default_branch":"master","permissions":{"admin":false,"push":false,"pull":false},"organization":{"login":"packit-service","id":46870917,"node_id":"MDEyOk9yZ2FuaXphdGlvbjQ2ODcwOTE3","avatar_url":"https://avatars3.githubusercontent.com/u/46870917?v=4","gravatar_id":"","url":"https://api.github.com/users/packit-service","html_url":"https://github.com/packit-service","followers_url":"https://api.github.com/users/packit-service/followers","following_url":"https://api.github.com/users/packit-service/following{/other_user}","gists_url":"https://api.github.com/users/packit-service/gists{/gist_id}","starred_url":"https://api.github.com/users/packit-service/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/packit-service/subscriptions","organizations_url":"https://api.github.com/users/packit-service/orgs","repos_url":"https://api.github.com/users/packit-service/repos","events_url":"https://api.github.com/users/packit-service/events{/privacy}","received_events_url":"https://api.github.com/users/packit-service/received_events","type":"Organization","site_admin":false},"network_count":12,"subscribers_count":9}'
+https://api.github.com/app/installations/1210186/access_tokens:
+- json:
+    expires_at: '2019-09-02T19:04:22Z'
+    permissions:
+      contents: read
+      issues: write
+      metadata: read
+      pull_requests: write
+      statuses: write
+    repository_selection: all
+    token: v1.4ef3073a1dd645c4137e15ce88031d7f84e14476
```

Comment:
Is that OK even if the token already expired?

> expires_at: '2019-09-02T19:04:22Z'

Reply:
Is there some other potential security leak?

Reply:
Only problem that comes to my mind right now is that someone will record new sessions and open PR while the token will be still valid.